src/components/ChipList/ChipList.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useLayoutEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import { useInView } from 'react-intersection-observer';
+import { withStyles } from '@material-ui/core/styles';
+import { Chip, Typography } from '@material-ui/core';
+
+const styles = {
+  chipWrapper: {
+    maxHeight: 80,
+    overflow: 'hidden',
+  },
+  chip: {
+    marginBottom: 8,
+    marginRight: 4,
+  },
+};
+
+const ChipList = ({ chips, classes }) => {
```

Comment:
The logic here is : 
1. Display only two lines chips by setting a maxHeight on the parent. 
2. Hide the other chips using `overflow: hidden` on the parent. 
3. Add an intersectionobserver to each chip
4. After rendering in `useLayoutEffect` figure out how many are not in view and set that in state.
5. Use that state variable to show `+ count more` 

I couldn't think of another way of doing this but open to suggestions.

Reply:
Not sure if you saw this section in material UI: https://material-ui.com/components/hidden/
Specifically like this: https://material-ui.com/components/hidden/#BreakpointDown.js

Reply:
I did not see that. I will see if I can change it in a followup pr.